boefjes/boefjes/plugins/kat_testssl_sh_ciphers/normalize.py

```diff
@@ -38,12 +38,14 @@ def parse_cipher(cipher: dict) -> dict | None:
             )
 
         return cipher_dict
+    else:
+        return None
 
 
 def run(input_ooi: dict, raw: bytes) -> Iterable[NormalizerOutput]:
     ip_service_reference = Reference.from_str(input_ooi["primary_key"])
     output = json.loads(raw)
-    tls_dict = {}
+    tls_dict: dict = {}
```

Comment:
It's already implicitly a dict type, so this is unnecessary to type hint?

Reply:
Mypy isn't able to figure out the type of the keys and values, so you get the following error:

```
boefjes/boefjes/plugins/kat_testssl_sh_ciphers/normalize.py:48: error: Need type annotation for "tls_dict" (hint: "tls_dict: Dict[<type>, <type>] = ...")  [var-annotated]
```

When looking at the code I saw that the types can more strict, so I've pushed a change for that.